34202.7 <c-11#183> Created logger
34202.7 <c-11#183:7> Logger was updated
34202.7 <c-11#183:7> Started intializing cube lib
34202.7 <c-11#183:7> Finished intializing cube lib with worker
34202.7 <c-11#183:7> shareCubes is called
34202.7 <c-11#183:7> Added 0 failed local cubes
34202.7 <c-11#183:7> Pruned 0 local cubes
34202.7 <c-11#183:7> CubeSolverThread entered cube retrieval loop, currently there are 0 cubes
34202.7 <c-11#183:7> returnFailedAndRequestCubes => [38] {}
34202.7 <c-11#183:7> Sent 0 failed cubes to root and requested new cubes
34202.7 <c-11#183:7> Received 4 cubes from root
34202.7 <c-11#183:7> CubeSolverThread entered cube retrieval loop, currently there are 4 cubes
34202.7 <c-11#183:7> CubeSolverThread retrieved 4 cubes
34202.7 <c-11#183:7> Started solving a cube
34202.7 <c-11#183:7> Cube failed
34202.7 <c-11#183:7> Added failed cube
34202.7 <c-11#183:7> Started solving a cube
34232.4 <c-11#183:7> Cube failed
34232.4 <c-11#183:7> Added failed cube
34232.4 <c-11#183:7> Started solving a cube
34232.9 <c-11#183:7> Cube failed
34232.9 <c-11#183:7> Added failed cube
34232.9 <c-11#183:7> Started solving a cube
34352.8 <c-11#183:7> appl_interrupt was called
34352.8 <c-11#183:7> appl_interrupt was called
34352.8 <c-11#183:7> appl_withdraw was called
34352.8 <c-11#183:7> Started withdraw thread
34352.8 <c-11#183:7> STOP (0.00s since last cb)
34352.8 <c-11#183:7> Solving interrupted
34352.8 <c-11#183:7> Leaving the main loop
34352.8 <c-11#183:7> Enter destructor of CubeWorkerInterface
34352.8 <c-11#183:7> Enter destructor of CubeRoot
34352.8 <c-11#183:7> Finished withdraw thread
34414.6 <c-11#183:7> Enter destructor
34414.6 <c-11#183:7> Time working: 150.113
34414.6 <c-11#183:7> Time suspended: 0.000
34414.6 <c-11#183:7> Joined cleanup thread
34414.6 <c-11#183:7> Exit destructor
34414.6 <c-11#183:7> Destructing logger
